### ISSUE TITLE
Hi,excuse me,I want to konw why the SI-SNR loss doesn't decrease when I used reverberation dataset?

### ISSUE THREAD

**Issue by yyd19948** [seorim0/DCCRN-with-various-loss-functions#8]

**Comment by seorim0** [comment]
Hi.
Could you please explain a little more?
I don't know the correlation between using reverberation dataset and not reducing the SI-SNR loss.

**Comment by yyd19948** [comment]
When the training data has no reverberation, the SI-SNR can be reduced to -17, but after adding the reverberation to the training data, it will be reduced to 6 and will not move.

**Comment by yyd19948** [comment]
> When the training data has no reverberation, the SI-SNR can be reduced to -17, but after adding the reverberation to the training data, it will be reduced to 6 and will not move.

When the training data has no reverberation, the SI-SNR can be reduced to -17, but after adding the reverberation to the training data, it will be reduced to 6 and will not move.

**Comment by seorim0** [comment]
The loss range may differ because the training dataset has become more difficult for DNN to learn as reverberation is added to the training dataset.
What was the result of the training? Was the model not able to reduce noise or reverberation?

**Comment by lixinlun** [comment]
it is not easy to answer his question so far, but i do remember, that the train-dataset in the original paper DCCRN(by tan) has reverberation.
"In detail, at each training epoch, we rst convolve speech and noise with a room impulse response (RIR) randomly-selected from a simulated 3000-RIR set by the image method [32], and then the speech-noise mixtures are generated dynamically by mixing reverb speech and noise at random SNR between -5 and 20 dB."

yyd1994 may change the train-dataset very slowly and careful and observe the difference, which might be a big project...........
